glamox/src/Components/NavbarDropdown.tsx

```diff
@@ -1,406 +0,0 @@
-import {
-  Popover,
-  PopoverTrigger,
-  PopoverContent,
-  PopoverHeader,
-  PopoverBody,
-  PopoverArrow,
-  PopoverCloseButton,
-  Button,
-  Tabs,
-  TabList,
-  Tab,
-  TabPanels,
-  TabPanel,
-  Flex,
-  CardBody,
-  TableContainer,
-  Table,
-  Tbody,
-  Tr,
-  Td,
-  Th,
-  Text,
-  Heading,
-} from "@chakra-ui/react";
-import { useNavigate } from "react-router-dom";
-
-export const NavbarDropdown = () => {
-  const navigate = useNavigate();
-
-  const dropdownData = [
```

Comment:
Never put your data inside your component. Always put that data outside your component or in a new file and import from there.